__tests__/helpers/getReadingTime.test.js

```diff
@@ -1,11 +1,11 @@
 import { getReadingTime } from '../../helpers';
-import { createArticle } from '../mocks/db.json';
+import { bodyDraftJs } from '../mocks/db.json';
 
 describe('getReadingTime', () => {
   test('Should return reading time', () => {
     expect.assertions(1);
     expect.stringContaining('min');
-    const readingTime = getReadingTime(createArticle.body);
+    const readingTime = getReadingTime(JSON.stringify(bodyDraftJs));
```

Comment:
Why are you changing the createArticle body into a JSON object?

Reply:
This is the main reason to come back to the backend. We are no longer receiving a text from the body of the article but an object which needs to be stringified to be saved in our database